test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -457,3 +470,24 @@ func mustParse(cpus string) *cpuset.CPUSet {
 	Expect(err).ToNot(HaveOccurred(), "failed to parse cpuset; cpus=%q", cpus)
 	return &set
 }
+
+func fetchSharedCPUsFromEnv(c *kubernetes.Clientset, p *corev1.Pod, containerName string) *cpuset.CPUSet {
+	GinkgoHelper()
+	// we should check the environment variable under the init process (pid 1)
+	cmd := []string{
+		"/bin/cat",
+		"/proc/1/environ",
```

Comment:
nice idea. I was thinking about changing the base image to dump the env, but this is even better